doc/api/fs.md

```diff
@@ -378,7 +378,8 @@ fs.appendFile('message.txt', 'data to append', 'utf8', callback);
 
 Any specified file descriptor has to have been opened for appending.
 
-_Note: Specified file descriptors will not be closed automatically._
+_Note: If a file descriptor is specified as the `file`, it will not be closed 
+automatically._
```

Comment:
That’s what my “LGTM” (= looks good to me) basically says, yes. :) It’s still part of the process of merging PRs that collaborators have the chance to state their opinion on changes, like @thefourtheye did, and discuss them if necessary.

Also, you could do us a favour by squashing the commits here together and formatting the commit message as requested by the [CONTRIBUTING.md](https://github.com/nodejs/node/blob/master/CONTRIBUTING.md#step-3-commit) (that’s also something that can be done by the person landing this PR).